src/main/java/com/boardgamesworld/bgrental/sort/infrastructure/api/BoardGameSortController.java

```diff
@@ -0,0 +1,43 @@
+package com.boardgamesworld.bgrental.sort.infrastructure.api;
+
+import com.boardgamesworld.bgrental.boardgame.domain.BoardGame;
+import com.boardgamesworld.bgrental.boardgame.domain.BoardGameCategory;
+import com.boardgamesworld.bgrental.boardgame.domain.BoardGameType;
+import com.boardgamesworld.bgrental.boardgame.infrastructure.api.BoardGameDto;
+import com.boardgamesworld.bgrental.boardgame.infrastructure.api.BoardGameMapper;
+import com.boardgamesworld.bgrental.boardgame.infrastructure.api.BoardGameResponse;
+import com.boardgamesworld.bgrental.sort.domain.BoardGameSortFacade;
+import com.boardgamesworld.bgrental.sort.domain.BoardGameSortType;
+import com.boardgamesworld.bgrental.sort.domain.InvalidSortException;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/boardgames/test")
+public class BoardGameSortController {
+
+    private final BoardGameSortFacade boardGameSortFacade;
+
+    @GetMapping
+    public ResponseEntity<BoardGameResponse> getSortedBoardGames(@RequestParam(value = "sort", required = false) BoardGameSortType sort,
```

Comment:
tak jak wspomniałem. Najwygodniej jest tutaj zrobic String sort

i na samym poczatku zmapować to na enuma, jak sie nie uda (nieznana wartość) to zwrócić błąd.